src/app/modules/main/wallet_section/add_account/internal/display_seed_phrase_state.nim

```diff
@@ -0,0 +1,15 @@
+type
+  DisplaySeedPhraseState* = ref object of State
+
+proc newDisplaySeedPhraseState*(backState: State): DisplaySeedPhraseState =
+  result = DisplaySeedPhraseState()
+  result.setup(StateType.DisplaySeedPhrase, backState)
+
+proc delete*(self: DisplaySeedPhraseState) =
+  self.State.delete
+  
+method executeCancelCommand*(self: DisplaySeedPhraseState, controller: Controller) =
```

Comment:
could we have some sort of inheritance for the cancel and delete which seems duplicated?

Reply:
All those states inherit the base `State` class. That gives us possibility to execute some special code, if needed, for each state. It's a coincidence that all states share the same logic for executing cancel command here in this case, so we could move it to base `State` class, but I somehow wanted to keep `State` as pure as possible. Anyhow... I can move the logic to the base class.

Reply:
@alaibe updated.